examples/resources/bloxone_dns_server/resource.tf

```diff
@@ -0,0 +1,14 @@
+
+resource "bloxone_dns_server" "example_all_server" {
+  name = "example_dns_sever"
+}
+
```

Comment:
you don't need this, the second one is enough

Reply:
done